httpdocs/js/webots-cloud.js

```diff
@@ -628,13 +632,17 @@ document.addEventListener('DOMContentLoaded', function() {
           if (data.error)
             ModalDialog.run('Project listing error', data.error);
           else {
+            if (project.email ? project.email.endsWith('@cyberbotics.com') : false)
```

Comment:
That works but is little bit odd, we are more used to this form:
```suggestion
            if (project.email && project.email.endsWith('@cyberbotics.com'))
```